include/openthread/instance.h

```diff
@@ -236,6 +236,28 @@ const char *otGetVersionString(void);
  */
 const char *otGetRadioVersionString(otInstance *aInstance);
 
+/**
+ * Function pointer used to set external CAlloc function for OpenThread.
```

Comment:
Function pointer should also be documented similar to a function. Please add a description of the parameters it takes. Same for one below.